bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -98,12 +101,13 @@ public Builder withRequest(ReadRowsRequest request) {
     Builder withKeys(byte[] startKey, byte[] stopKey) {
       final ByteString start = ByteStringer.wrap(startKey);
       final ByteString stop = ByteStringer.wrap(stopKey);
-      request =
-          request.toBuilder()
+      ReadRowsRequest request = this.request == null
+          ? ReadRowsRequest.getDefaultInstance()
+          : this.request.get();
```

Comment:
get() cannot be called at runtime.

Reply:
correction: get() cannot be called at construction time.

Reply:
This method gets called at runtime only.  Please notice that it's package private.  It's called for splitting purposes.